MapleServer2/PacketHandlers/Game/GuildHandler.cs

```diff
@@ -117,15 +117,14 @@ private void HandleCreate(GameSession session, PacketReader packet)
         {
             string guildName = packet.ReadUnicodeString();
 
-            if (session.Player.Wallet.Meso.Amount < 2000)
+            if (!session.Player.Wallet.Meso.Modify(-2000))
             {
                 short NoticeCode = 5121;
                 session.Send(GuildPacket.ErrorNotice(NoticeCode));
                 return;
             }
             else
```

Comment:
Since you're returning, you can safely remove this else and unindent the following code